src/ui/components.tsx

```diff
@@ -150,7 +154,7 @@ export const CheckMkSelectNegatable = <T extends RequestSpecNegatableOptionKeys>
   const value: NegatableOption =
     props.value === undefined
       ? {
-          value: '',
```

Comment:
why was this changed? have you tested it?

Reply:
It was the change that got lost during commit splitting I talked about. I tested it and now it displays the placeholder if no value was passed as a prop.

Reply:
I'll do that